commit/core.pyx

```diff
@@ -652,13 +652,25 @@ cdef class Evaluation :
             If None (default), they will end up in the .pyxbld directory in the user’s home directory.
             If using this option, it is recommended to use a temporary directory, quit your python 
                 console between each build, and delete the content of the temporary directory.
+        tikhonov_equalizer: float
+            equalizer parameter of the Tikhonov regularization term
+        deriv_matrix: string
+            derivative matrix of the Tikhonov regularization term
+            If None (default), no regularization term is added to the model
+            If using this option, tikhonov_equalizer must be positive
```

Comment:
Why? This term should not be considered in the optimization. I imagine an experiment evaluating different values, and the typical scenario is to test for lambda equal 0, 1e-6, etc

Reply:
Positive lambda values are supported, but they require always a L matrix. I set an IF statement which verifies that L is not None when a positive lambda value is given.